k8s/prom-operator.yml

```diff
@@ -143,6 +143,9 @@ prometheus:
           topologyKey: "kubernetes.io/hostname"
     nodeSelector:
       testground.nodetype: infra
+    podMonitorSelector:
```

Comment:
Doesn't seem to be doing the trick :( I still see testplan pods being scraped...